bin/iex

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker-compose -f ../docker-compose.yml exec reticulum iex $*
```

Comment:
Might be good to add a README to this `bin/` directory to explain what these scripts are, and how they should be used. Maybe it would just be one sentence, with a link to the compose repo.

Reply:
Done. Will add a link to the Hubs Compose repo once that is up.